src/sentry/metrics/composite_experimental.py

```diff
@@ -0,0 +1,57 @@
+from typing import Any, Dict, Optional, Type, Union
+
+from sentry.metrics.base import MetricsBackend, Tags
+from sentry.metrics.dummy import DummyMetricsBackend
+from sentry.metrics.minimetrics import MiniMetricsMetricsBackend
+from sentry.utils.imports import import_string
+
+
+class CompositeExperimentalMetricsBackend(MetricsBackend):
+    def __init__(self, prefix: Optional[str] = None, **kwargs: Any):
```

Comment:
Let's give this an allow list option that selectively sends metrics into the minimetrics backend.

Reply:
Will do!